File: 1019-SquaresOfASortedArray/1019-SquaresOfASortedArray.js
// Last updated: 3/6/2026, 2:09:04 PM
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var sortedSquares = function(nums) {
    let res = [];
    for(let i=0;i<nums.length;i++){
        res.push(nums[i]*nums[i]);
    }
    res=res.sort((a,b)=>a-b);
    return res;
};

File: 1019-SquaresOfASortedArray/1019-SquaresOfASortedArray.cpp
// Last updated: 3/6/2026, 2:09:06 PM
class Solution {
public:
    vector<int> sortedSquares(vector<int>& nums) {
        vector<int> res;

        for(int x:nums){
            res.push_back(x*x);
        }
        sort(res.begin(),res.end());
        return res;
    }
};